AEPIdentity/Sources/Identity+PublicAPI.swift

```diff
@@ -112,7 +112,7 @@ import Foundation
     static func syncIdentifiers(identifiers: [String: String]?, authenticationState: MobileVisitorAuthenticationState) {
         var eventData = [String: Any]()
         eventData[IdentityConstants.EventDataKeys.IDENTIFIERS] = identifiers
-        eventData[IdentityConstants.EventDataKeys.AUTHENTICATION_STATE] = authenticationState.rawValue
+        eventData[IdentityConstants.EventDataKeys.AUTHENTICATION_STATE] = authenticationState
```

Comment:
Setting the `MobileVisitorAuthenticationState` in the event data causes the logged event to just display the enum name instead of its value:
```
data:  {
    authenticationstate = "AEPIdentity.MobileVisitorAuthenticationState";
    forcesync = 0;
    issyncevent = 1;
    visitoridentifiers =     {
        idType1 = 1234567;
    };
```
An alternative to this fix is to continue to set the raw value in the Event but change Event+Identity to cast the value as an Int and then convert to a MobileVisitorAuthenticationState when reading the event data.